packages/core-mobile/e2e/tests/notification/sendNotification.e2e.ts

```diff
@@ -0,0 +1,36 @@
+import { warmup } from '../../helpers/warmup'
+import sendLoc from '../../locators/send.loc'
+import accountManagePage from '../../pages/accountManage.page'
+import notificationsPage from '../../pages/burgerMenu/notifications.page'
+import portfolioPage from '../../pages/portfolio.page'
+import sendPage from '../../pages/send.page'
+
+describe('Send Notification', () => {
```

Comment:
The push notification is broken currently. I added a file and folder as a placeholder and a reminder for future work. I marked the test to be ignored in config files.